scripts/hail_batch/increase_snp_chip_partitions/README.md

```diff
@@ -0,0 +1,9 @@
+# Project WGS samples onto SNP-chip PCA
+
+This runs a Hail query script in Dataproc using Hail Batch in order to increase the number of partitions in the SNP-chip matrix table. Since the matrix table currently only has one partition, this results in a high amount of work done using once process only. This is particularly difficult when intersecting with another matrix table that has many partitions. To run, use conda to install the analysis-runner, then execute the following command:
```

Comment:
```suggestion
This runs a Hail query script in Dataproc using Hail Batch in order to increase the number of partitions in the SNP-chip matrix table. Since the matrix table currently only has one partition, this results in a high amount of work done using one worker only. This is particularly difficult when intersecting with another matrix table that has many partitions. To run, use conda to install the analysis-runner, then execute the following command:
```